chains/zksyncsepolia/metadata.yaml

```
@@ -0,0 +1,19 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://explorer.sepolia.era.zksync.dev/contract_verification 
+    family: zksync
```

`zksync` is not a supported explorer family type yet
```suggestion
    family: etherscan
```